### PR TITLE
Fix unmount crash

### PR DESCRIPTION
**Pull request by mdnsk**

### CODE REVIEW

File: src/react-aria-modal.js [hunk 1]
```diff
@@ -268,39 +268,31 @@ function displace(WrappedComponent, options) {
...
       }
 
-      this.setState({ container });
+      this.forceUpdate();
     }
 
     componentWillUnmount() {
       if (!options.renderTo) {
-        this.state.container.parentNode.removeChild(this.state.container);
+        this.container.parentNode.removeChild(this.container);
```
Comment (mdnsk) [inline src/react-aria-modal.js:286]: In rare cases, it's possible that `componentWillUnmount` will be called before `this.setState({ container })` applied changes to state. So it crashes when trying access `parentNode` property of `containter` state which is null.

I have replaced state by class property and added invocation of `this.forceUpdate()` in order to rerender component after container was mounted to DOM.

**Comment by mdnsk** [comment]
@tristen, have a look on this MR, please :)

**Review by tristen — APPROVED** [review]

**Comment by mdnsk** [comment]
@tristen, there is still old files in `dist` folder in `react-aria-modal@5.0.1` 🤔 

```js
    {
      key: "componentWillUnmount",
      value: function componentWillUnmount() {
        if (!options.renderTo) {
          this.state.container.parentNode.removeChild(this.state.container);
        }
      }
    }
```

Looks like `prepublish` script hasn't been run before publishing a new version.

**Comment by tristen** [comment]
Ahh thanks @mdnsk. Looks like `prepublish` is deprecated. Will fix!

**Pull request merged**